### ISSUE TITLE
Script bloqué sur  "Checking folder size..."

### ISSUE THREAD

**Issue by cooljazz** [warezcmpt/JRabbitFreeboxSync#1]
Bonjour, 
Merci pour le script. 
Je block sur  "Checking folder size..."

En lftp j'ai la même erreur que sur le post : 
https://mondedie.fr/d/8316-Iptables-ftp-tls-ftpes/9

Récupération du contenu du répertoire (0) [En attente de la connexion de données...]

Pourtant iptable est désactivé, il n'y a pas de Firewall ?

As-tu une idée ?

**Comment by warezcmpt** [comment]
Deja le ftp "extérieur" est activé?

Sur mafreebox.fr:
Patage de fichier/FTP
Activer l'acces a distance

Ex de l'adresse ftp a entrer:
ftpes://freebox@83.853.160.134:3452/

**Comment by cooljazz** [comment]
Oui, cela fonctionne à partir d'un serveur Windows Server 2012 R2 avec Filezilla. 
J'aimerai que cela fonctionne en lftp à partir d'un serveur Debian Jessie.

**Comment by warezcmpt** [comment]
Je me rappel pas trop mais essaye  de supprimer # devant:
#set ftp:port-range $Home_ftp_port
#set ftp:passive-mode false
Et redémarre le script
